src/lib/error/ErrorReport.ts

```diff
@@ -140,7 +144,29 @@ export class ErrorReport extends Error {
       stack: this.stack,
       env: this.env,
       ...(this.originalError.code ? { code: this.originalError.code } : null),
+    })
+  }
+
+  private truncateStringsFromObject(element: any, maxStrSize: number = ErrorReport.MAX_ERROR_STRING_LENGTH) {
```

Comment:
This can be a `public static` so we can create tests for it in the future - or even a new file exporting this function, what do you think?